backend/src/controllers/game_controller.spec.ts

```diff
@@ -125,7 +125,7 @@ describe("Test POST, PUT, GET", () => {
       .send();
     expect(res.status).toEqual(200);
     expect(res.body.id).toBe(game.id);
-    expect(res.body.startDate).toBe("2020-01-21T15:55:25.436Z");
+    expect(typeof res.body.startDate).toEqual("string");
```

Comment:
I do not believe this is a sufficiant test to know if it is a date. The old version was better even if it wasn't optimal.

Reply:
On pourrais faire 
```
expect(/^\d{4}-[0-1]\d-[0-3]\dT(\d{2}:?){3}.\d{3}Z$/.test(res.body.startDate)).toBeTruthy();
```